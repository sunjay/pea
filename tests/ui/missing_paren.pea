fn main() {
    println!b"hello, world!");
}
