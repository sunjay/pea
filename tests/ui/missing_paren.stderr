[tests/ui/missing_paren.pea:2:13-2:28] error: expected `(`, found: a byte string literal

error: aborting due to 1 previous error

